### ISSUE TITLE
Play button not changing to pause

### ISSUE THREAD

**Issue by JosephSilber** [Selz/plyr#129]
It works in the demos, but I can't get it to work. There's no pause button when the audio is playing.

Here's what I've got: http://josephsilber.com/demos/plyr-test/

**Comment by SamPotts** [comment]
Are you using the `develop` branch? That's not really production ready code at this point.

**Comment by JosephSilber** [comment]
It happens to me on the master branch as well, but I haven't been able to produce a reduced test case yet.

I'm trying to debug it in my app. I can't figure out where it actually happens in the code.

[All I see here](https://github.com/Selz/plyr/blob/master/src/js/plyr.js#L1754) is that if focuses on the pause button. Where does it toggle the visibility of the buttons?

**Comment by JosephSilber** [comment]
The problem was that since I wasn't using the standard `.player` class in the HTML, the CSS wasn't applied properly, since it uses the parent as the selector for the buttons.

**Comment by SamPotts** [comment]
Ahhh it makes sense. I'll add something into the docs about that for v1.5.

**Comment by JosephSilber** [comment]
Just to be clear: this is not exclusive to the `develop` branch. After you said I shouldn't use the `develop` branch, I switched it out for the `master` branch with this:

```js
plyr.setup({
    selectors: { container: '.my-custom-class' }
});
```

This obviously had the same effect, so this issue is not exclusive to the `develop` branch.

**Comment by SamPotts** [comment]
Fixed in [v1.5.0](https://github.com/Selz/plyr/releases/tag/v1.5.0). `plyr` is now exclusively the namespace for CSS.
